solutions/usaco-491.mdx

```diff
@@ -10,6 +10,71 @@ author: Qi Wang
 **Time Complexity:** $\mathcal{O}(N)$
 
 <LanguageSection>
+	
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+unordered_map<int, vector<int>> mp;
```

Comment:
bad practice, why not just an array of vectors?
also `mp` doesnt really make sense, maybe name it `adj` or `graph`?